src/App.js

```diff
@@ -9,6 +9,7 @@ function App() {
     <div className="App">
       <Routes>
         <Route path='/articles' element={<ArticleList />}/>
```

Comment:
You don't have a route for the path `/` when you should have. This should be your home where you can navigate your app.